src/Initializer/Parameters/OutputParameters.cpp

```diff
@@ -88,6 +76,7 @@ EnergyOutputParameters readEnergyParameters(ParameterReader* baseReader) {
   const auto computeVolumeEnergiesEveryOutput =
       reader->readWithDefault("computevolumeenergieseveryoutput", 1);
   const auto terminalOutput = reader->readWithDefault("energyterminaloutput", false);
+  const auto termainlPrecision = reader->readWithDefault("energyterminalprecision", 6);
```

Comment:
Sorry to nitpick but the variablename seems to have a typo

Reply:
Those are the best usually—fixed.

(complaining about typos is completely fine—they're also bugs imo)